src/components/PrivacyAgent.js

```diff
@@ -0,0 +1,58 @@
+class PrivacyAgent { // eslint-disable-line no-unused-vars
+    /**
+     * @param {string} nexthash - the hash to route to when the user clicked "ok"
+     * @param {HTMLElement} [el]
+     */
+    constructor(nexthash = 'continue', el) {
```

Comment:
Better to use `Nimiq.Observable` and to `fire` an event when the user clicks the "accept" button, like other components. The component should not be required to know about the environment that it is used in.